From 35de6babe3c2ed04885f05c6944860dafdaf2505 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 30 Dec 2020 17:46:40 +0100
Subject: [PATCH 1/1] doc: android/boot-image: invalid C declaration

make htmldocs results in an error:

doc/android/boot-image.rst:33:
WARNING: Unparseable C cross-reference: 'struct andr_img_hdr'
Invalid C declaration: Expected identifier in nested name, got keyword:
struct [error at 6]

Follow the style prescribed in
https://www.kernel.org/doc/html/latest/doc-guide/kernel-doc.html#highlights-and-cross-references

Add missing definite article.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 doc/android/boot-image.rst | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/doc/android/boot-image.rst b/doc/android/boot-image.rst
index 195da688f4..8648ab721b 100644
--- a/doc/android/boot-image.rst
+++ b/doc/android/boot-image.rst
@@ -30,9 +30,10 @@ next image headers:
 
 v2, v1 and v0 formats are backward compatible.
 
-Android Boot Image format is represented by :c:type:`struct andr_img_hdr` in
-U-Boot, and can be seen in ``include/android_image.h``. U-Boot supports booting
-Android Boot Image and also has associated command
+The Android Boot Image format is represented by
+:c:type:`struct andr_img_hdr <andr_img_hdr>` in U-Boot, and can be seen in
+``include/android_image.h``. U-Boot supports booting Android Boot Image and also
+has associated command
 
 Booting
 -------
-- 
2.29.2

